### PR TITLE
modify origin-aggregated-logging test targets

### PR DESCRIPTION
**Pull request by jcantrill**
Modify OAL testing targets

### CODE REVIEW

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml [hunk 1]
```diff
@@ -74,15 +74,7 @@ tests:
...
-  commands: |
-    . openshift/ci-operator/build-image/setup-logging-for-e2e.sh
-  openshift_installer_src:
-    cluster_profile: aws
-- as: upgrade-full-integ-aws
-  commands: |
-    OPENSHIFT_BUILD_NAMESPACE= IMAGE_FORMAT= USE_CUSTOM_IMAGES=true hack/deploy-logging.sh
-    export LOGGING_DEPLOY_MODE=upgrade
-    . openshift/ci-operator/build-image/setup-logging-for-e2e.sh
+- as: e2e
```
Comment (ewolinetz) [inline ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: this is our `smoke test` currently right?
shouldn't we name it as such, or is the idea we'll eventually be building that up more in the future?

Comment (jcantrill) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: I would expect we might want to add more tests, but I'm open to changing it.  the kickoff script I called e2e and thus the reason i chose this name

Comment (ewolinetz) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: i think its fine... though it does give us an opportunity to have a generic `smoke` entry point and then an `e2e` separate so we can determine more easily if its an issue with the image build/components standing up or how the components are being stressed

**Comment by ewolinetz** [comment]
```
F0219 15:20:05.292752     354 helpers.go:114] Error from server (NotFound): namespaces "openshift-logging" not found
goroutine 1 [running]:
k8s.io/klog.stacks(0xc00105b500, 0xc000ddc1a0, 0x75, 0xc6)
	/go/src/github.com/openshift/oc/vendor/k8s.io/klog/klog.go:875 +0xb8
k8s.io/klog.(*loggingT).output(0x4e2efe0, 0xc000000003, 0xc0007d11f0, 0x49ca2d2, 0xa, 0x72, 0x0)
	/go/src/github.com/openshift/oc/vendor/k8s.io/klog/klog.go:826 +0x330
k8s.io/klog.(*loggingT).printDepth(0x4e2efe0, 0x3, 0x2, 0xc0012d3928, 0x1, 0x1)
	/go/src/github.com/openshift/oc/vendor/k8s.io/klog/klog.go:698 +0x129
k8s.io/klog.FatalDepth(...)
	/go/src/github.com/openshift/oc/vendor/k8s.io/klog/klog.go:1262
k8s.io/kubectl/pkg/cmd/util.fatal(0xc0003d0730, 0x46, 0x1)
	/go/src/github.com/openshift/oc/vendor/k8s.io/kubectl/pkg/cmd/util/helpers.go:92 +0x1d0
k8s.io/kubectl/pkg/cmd/util.checkErr(0x325d1c0, 0xc000d46500, 0x2ebc490)
	/go/src/github.com/openshift/oc/vendor/k8s.io/kubectl/pkg/cmd/util/helpers.go:175 +0x95d
k8s.io/kubectl/pkg/cmd/util.CheckErr(...)
	/go/src/github.com/openshift/oc/vendor/k8s.io/kubectl/pkg/cmd/util/helpers.go:114
k8s.io/kubectl/pkg/cmd/get.NewCmdGet.func1(0xc0011a1180, 0xc001086ea0, 0x1, 0x9)
	/go/src/github.com/openshift/oc/vendor/k8s.io/kubectl/pkg/cmd/get/get.go:167 +0x151
github.com/spf13/cobra.(*Command).execute(0xc0011a1180, 0xc001086e10, 0x9, 0x9, 0xc0011a1180, 0xc001086e10)
	/go/src/github.com/openshift/oc/vendor/github.com/spf13/cobra/command.go:830 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc001104500, 0x2, 0xc001104500, 0x2)
	/go/src/github.com/openshift/oc/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/go/src/github.com/openshift/oc/vendor/github.com/spf13/cobra/command.go:864
main.main()
	/go/src/github.com/openshift/oc/cmd/oc/oc.go:110 +0x853 
```

File: ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml (hunk 1 above)
Comment (jcantrill) [reply ci-operator/config/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master.yaml]: updated to 'smoke' per slack convo

**Pull request merged**